docs/fsharp/language-reference/query-expressions.md

```diff
@@ -47,7 +47,7 @@ query1
 |> Seq.iter (fun customer -> printfn "Company: %s Contact: %s" customer.CompanyName customer.ContactName)
 ```
 
-In the previous code example, the query expression is in curly braces. The meaning of the code in the expression is, return every customer in the Customers table in the database in the query results. Query expressions return a type that implements `System.Linq.IQueryable&#96;1` and `System.Collections.Generic.IEnumerable&#96;1`, and so they can be iterated using the [Seq module](https://msdn.microsoft.com/library/54e8f059-ca52-4632-9ae9-49685ee9b684) as the example shows.
+In the previous code example, the query expression is in curly braces. The meaning of the code in the expression is, return every customer in the Customers table in the database in the query results. Query expressions return a type that implements [IQueryable<T>](https://docs.microsoft.com/dotnet/core/api/system.linq.iqueryable-1) and [IEnumerable<T>](https://docs.microsoft.com/dotnet/core/api/system.collections.generic.ienumerable-1), and so they can be iterated using the [Seq module](https://msdn.microsoft.com/library/54e8f059-ca52-4632-9ae9-49685ee9b684) as the example shows.
```

Comment:
I meant: "name-of-the-folder/name-of-the-file.md" ? Not sure where to find the path for those links :(